tes-adapter/src/main/java/com/epam/pipeline/controller/Controller.java

```diff
@@ -0,0 +1,12 @@
+package com.epam.pipeline.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class Controller {
```

Comment:
It is better to use mo specific names for your classes. Imagine that you have 20+ Controller classes, how you will navigate?